src/phantom.js

```diff
@@ -45,7 +36,36 @@ export default class Phantom {
         }
 
         let pathToShim = path.normalize(__dirname + '/shim.js');
-        logger.debug(`Starting ${phantomPath} ${args.concat([pathToShim]).join(' ')}`);
+
+        let loggerLevel = process.env.DEBUG === 'true' ? 'debug' : 'info';
+        if(typeof config.logger === 'object') {
```

Comment:
Okay, I think I see what you mean. However each time a `phantom` instance is created using the default logger with a different log level it would override the previous log level value (which is still better than before, when we had almost no way to change it at all :smile: ).